src/js/close-button.js

```diff
@@ -1,24 +1,45 @@
+/**
+ * @file close-button.js
+ */
 import Button from './button';
 import Component from './component';
 
-/**
- * The `CloseButton` component is a button which fires a "close" event
- * when it is activated.
- *
- * @extends Button
- * @class CloseButton
- */
 class CloseButton extends Button {
 
+  /**
+   * The `CloseButton` component is a button which fires a "close" event
+   * when it is activated.
+   *
+   * @param {Object} player
+   *        the player that this component should ataach to
+   *
+   * @param {Object} [options]
+   *        Object of option names and values
+   *
+   * @extends Button
+   */
   constructor(player, options) {
     super(player, options);
     this.controlText(options && options.controlText || this.localize('Close'));
   }
 
+  /**
+   * Returns the css attachments for this element
+   *
+   * @return {string}
+   *         The current css class for thes CloseButton
+   */
   buildCSSClass() {
     return `vjs-close-button ${super.buildCSSClass()}`;
   }
 
+  /**
+   * Triggers a close event when we are clicked,
+   * or focused and enter/space is pressed down
```

Comment:
I think we shouldn't use personal language here (i.e. "we are" -> "the component is"). Also, comma is unnecessary here.